src/components/OasisMarketHistory.jsx

```diff
@@ -1,18 +1,53 @@
 import React, { PureComponent } from 'react';
 import { PropTypes } from 'prop-types';
-// import ImmutablePropTypes from 'react-immutable-proptypes';
+import moment from 'moment';
+
+import OasisWidgetFrame from '../containers/OasisWidgetFrame';
+import { OasisTable } from './OasisTable';
+import { tradeType, price, formatTradeType, formatPrice, getBaseAndQuotePrice } from '../utils/tokens/pair';
+import { orderBytTimestamp, SORT_DESC } from '../utils/sort';
```

Comment:
1.You have typo in ..BytTimestamp
2. Since it's from the sort module , why not only `DESCENDING` or `DESC` - this is a suggestion

Reply:
Well, yeah but its not exactly clear, although its pretty simple to work this out though.

Reply:
Renamed to ASCENDING  & DESCENDING.